src/components/profile/Profile.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+const Profile = () => {
+  const joinMissions = useSelector((state) => state.mission);
+  const reserveRockets = useSelector((state) => state.post);
+  const filteredMissions = joinMissions.filter(
+    (mission) => mission.canceled === true
+  );
+  const reservedRockets = reserveRockets.filter(
+    (rocket) => rocket.canceled === true
+  );
+  return (
+    <div className="container d-flex justify-content-center my-5">
+      <div>
+        <h2>My Missions</h2>
+        <div className="card mission-card">
+          <ul className="list-group list-group-flush">
```

Comment:
- [OPTIONAL] The `My Profile` section can be enhanced by adding a placeholder message when the "My Missions" or "My Rockets" lists are empty (no missions joined or no rockets reserved).
- [OPTIONAL] The `My Profile` section can be enhanced by adding the "Cancel reservation" and "Leave Mission" buttons to the lists here. Clicking them should dispatch the actions you have already used in the main Rockets and Missions sections.
- [OPTIONAL] The `My Profile` section can be enhanced by adding the "Read more" button for each mission and rocket. Upon click, it should open a corresponding Wikipedia page in a new tab. NOTE - you need to get that extra Wikipedia URL from the API's payload.

Reply:
@BenMukebo Thank you so much for leaving me comments to improve my project. I will come back tomorrow and implement all the changes you suggested.